spyder/widgets/explorer.py

```diff
@@ -532,7 +532,22 @@ def delete_file(self, fname, multiple, yes_to_all):
                               "<br><br>Error message:<br>%s"
                               ) % (action_str, fname, to_text_string(error)))
         return False
-
+    
+    def delete_project(self, pname):
+        """Deletes the .spyproject diretory"""

```

Comment:
typo, diretory -> directory

dot at the end. and imperative mood ;-)

`"""Delete the .spyproject directory."""`